index.qmd

```diff
@@ -39,73 +40,58 @@ format:
           margin-left: 0
         embed-resources: true
         page-layout: full
-my-custom-stuff:
-   my-reuseable-variable: "I can use this wherever I want in the markdown, and change it in only once place :)"
 ---
 
-## Contents
 
-Some example slides - [also look at example RevealJS slides in the Quarto docs](https://quarto.org/docs/presentations/revealjs/)
+## Welcome
 
-* Non-executable and executable code-blocks
-* bullet points with highlighting 
-* two-column slides 
-* how to include a slide from a separate MD file
-* preview and link to a webpage
+![](img/sainsbury-wellcome-centre.jpg)
```

Comment:
I assume this image is here as a placeholder and we may eventuall put something else?

Reply:
Perhaps we can link to the "Event guide" google doc here.

Reply:
Also link to the main neuroinformatics unit OSW website.

Reply:
> is here as a placeholder and we may eventuall put something else?

Not necessarily, but happy to change. It felt generic and suitable to have up on screen as people arrive in the morning. No strong opinion.

Reply:
I was thinking that the title slide already plays that role, and it has the same image. If we change the title to "Welcome to NIU OSW" we can leave that up and don't need this one.